src/issues/index.js

```diff
@@ -2,20 +2,50 @@ import React from 'react'
 import { useIssuesDB } from './api.js'
 import { BrowserRouter as Router, Route } from 'react-router-dom'
 import { List } from './list'
+import Snackbar from '@material-ui/core/Snackbar';
+import MuiAlert from '@material-ui/lab/Alert';
 
 const issueLookup = (id) =>
   fetch(process.env.MARVEL_LOOKUP_URL + '/' + id).then((resp) => resp.json())
 
 export const Issues = () => {
-  const DEFAULT_UPC_CODE = '75960609709800311'
+	const DEFAULT_UPC_CODE = '75960609709800311'
+	const ISSUE_NOT_FOUND = 'Issue not found.'
   const { create, issues } = useIssuesDB()
-  const [newIssueId, setNewIssueId] = React.useState(DEFAULT_UPC_CODE)
+	const [newIssueId, setNewIssueId] = React.useState(DEFAULT_UPC_CODE)
+	const [newIssue, setNewIssue] = React.useState(null)
+	const [hasError, setHasError] = React.useState(false)
+	const [hasSuccess, setHasSuccess] = React.useState(false) 
 
   const createNewIssue = () => {
-    issueLookup(newIssueId)
-      .then((resp) => create({ ...resp, upc: newIssueId }))
-      .then(() => setNewIssueId(DEFAULT_UPC_CODE))
-  }
+		issueLookup(newIssueId)
+			.then((resp) => {
+				if(resp.data) {
+					// no issue found
+					throw ISSUE_NOT_FOUND;
+				} else {
+					// got an issue back
+					setNewIssue(resp);
+					setHasError(false);
+					return resp;
+				}
+			})
+			.then((resp) => create({ ...resp, upc: newIssueId }))
+			.then(() => {
+				setHasSuccess(true)
+				setNewIssueId(DEFAULT_UPC_CODE)
+			})
+			.catch(error => {
+				if(error === ISSUE_NOT_FOUND) {
+					setHasError(true);
+				}
+			})
+	}
+	
+	const closeErrorSnackbar = () => {
```

Comment:
I added this function so we could set 2 state properties. Is there a better way to set multiple state properties at once in order to avoid having to pull this out into a function? 
```suggestion
	const closeErrorSnackbar = () => {
```

Reply:
I'd probably call this function (or at least think about it) as "reset". Maybe `resetAddForm` or something.

Reply:
But no, that's the way to set multiple state variables 👍